.gitignore

```diff
@@ -29,23 +29,23 @@ $RECYCLE.BIN/
 # OSX
 # =========================
 
-.DS_Store
-.AppleDouble
-.LSOverride
-
-# Icon must ends with two \r.
-Icon

-
-# Thumbnails
-._*
-
-# Files that might appear on external disk
-.Spotlight-V100
-.Trashes
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must ends with two \r.
+Icon

```

Comment:
Removed the `^M` (`\r`)?

Reply:
Yes. Visual Studio did that on me. It looked like it had formatted the file and played with newlines.

Reply:
Even if VS did it for you, you should not have pushed that commit. You turned it from a special folder name `Icon\r` (an OSX thing) to a generic folder name `Icon`, which would cause any file or folder named "Icon" to be ignored, which can cause a headache for someone who assumes they safely committed their Icon folder.

The original commit added .vs, which is not a problem. It prevents Visual Studio's user-specific folder from being committed.

The second commit should be reverted.